### ISSUE TITLE
Query Expressions for JSON rfc9535

### ISSUE THREAD

**Issue by vincent-herlemont** [jmespath/jmespath.rs#53]
I am opening this issue to explore the possibility of including the ability to use Query Expressions. Was this planned? What do you think?

Link to the RFC: https://datatracker.ietf.org/doc/rfc9535/

**Comment by cetra3** [comment]
Yes, definitely open to it but have not got the bandwidth at the moment to implement!
